pkgs/tools/security/posteid-seed-extractor/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+}:
+
+python3Packages.buildPythonApplication {
+  pname = "posteid-seed-extractor";
+  version = "unstable-23-02-2022";
+
+  src = fetchFromGitHub {
+    owner = "simone36050";
+    repo = "PosteID-seed-extractor";
+    rev = "667e2997a98aa3273a6bf6b4b34ca77715120e7f";
+    hash = "sha256-smNwp67HYbZuMrl0uf2X2yox2JqeEV6WzIBp4dALwgw=";
+  };
+
+  format = "other";
+
+  nativeBuildInputs = [
+    python3Packages.wrapPython
+  ];
+
+  pythonPath = with python3Packages; [
+   certifi
+   cffi
+   charset-normalizer
+   cryptography
+   idna
+   jwcrypto
+   pycparser
+   pycryptodome
+   pyotp
+   qrcode
+   requests
+   urllib3
+   wrapt
+  ];
+
+  installPhase = ''
+    mkdir -p "$out/bin"
+    cp ./extractor.py "$out/bin/"
+    chmod +x "$out/bin/extractor.py"
```

`install` can do all that in one command, also let's drop the extension and use a less generic name since this will go into the user's PATH

```suggestion
    install -Dm755 extractor.py $out/bin/posteid-seed-extractor
```